.github/workflows/deployment.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish package to PyPI and TestPyPI
+on: push
```

Comment:
Will this happen on any push?

I think we might want to work out when we want to release a new version.